Sources/StreamChat/Config/StreamRuntimeCheck.swift

```diff
@@ -14,4 +14,9 @@ public enum StreamRuntimeCheck {
     ///
     ///  Enables background mapping of DB models
     public static var _isBackgroundMappingEnabled = false
+
+    /// Enable/Disable local filtering for Channel lists. When enabled,
+    /// the `ChannelDTO` will include the filter's predicate (if available)
+    /// in the fetchRequest.
+    public static var isChannelLocalFilteringEnabled = true
```

Comment:
I'm not sure this one is a candidate for `StreamRuntimeCheck` 🤔 We already have a `ChatClientConfig` which IMO it is a better candidate for this. CC @polqf @martinmitrevski

Reply:
I agree with Nuno here

Reply:
Great, I'll move it to the `ChatClientConfig` then